setup.py

```diff
@@ -313,5 +313,9 @@ def get_outputs(self):
     description="Transformer acceleration library",
     ext_modules=ext_modules,
     cmdclass={"build_ext": TEBuildExtension},
+    install_requires = [
+        "einops",
```

Comment:
I'm not 100% sure we need einops - in the end the 3 usages of rearrange we have are just glorified views...